docs/389ds/howto/howto-bind.md

```diff
@@ -57,7 +57,7 @@ To implement this solution, the following things need to happen:
         dnszonename: foobar.com
         dnsclass: IN
         dnstype: SOA
-        dnszonemaster: tim.foobar.com
+        dnszonesupplier: tim.foobar.com
```

Comment:
I have a doubt. Is dnszonemaster a valid term ?

Reply:
IMHO the whole howto-bind document is outdated, the links to ldap2dns now leads to sourceforge main page but  they do not have any project about ldap2dns ... 
IMHO we should rather left it as it and move it in an "Outdated" section ...